Resources/Prototypes/_Nyano/Markers/Spawners/jobs.yml

```diff
@@ -22,18 +22,6 @@
       - state: green
       - state: mailcarrier
 
-- type: entity
-  id: SpawnPointCyborg
-  parent: SpawnPointJobBase
-  name: cyborg
-  components:
-  - type: SpawnPoint
-    job_id: Cyborg
-  - type: Sprite
-    layers:
-      - state: green
-      - state: roboticist
-
```

Comment:
did you verify these were either removed from maps or you made a proper migration file?

Reply:
There were spawn points on 2 medical ships that I seen you remove, buy I can give it anther test.

Reply:
No errors